package-lock.json

```diff
@@ -8,6 +8,7 @@
       "name": "@lob/ui-components",
       "version": "1.0.0-beta.1",
       "dependencies": {
+        "@lob/ui-components": "^0.0.196",
```

Comment:
I think there's something wrong here, ui-components shouldn't depend on itself.  I think you should probably revert the changes to this file.